platform-sdk/swirlds-common/src/test/java/com/swirlds/common/wiring/model/ModelTests.java

```diff
@@ -1677,4 +1678,21 @@ void multipleSequentialSchedulerTest() {
 
         validateModel(model, false, true);
     }
+
+    @Test
+    void unboundInputWireTest() {
```

Comment:
Attn @alittley: in your platform wiring branch, I suggest adding a unit test that binds the platform wiring to mock implementations. Then do `assertFalse(platformWiring.getModel().checkForUnboundInputWires()));`, thus preventing future branches from creating inputs without properly binding them.

Reply:
Noted - I'll add it once this PR merges